### PR TITLE
Improve grammar and spelling

### PR DESCRIPTION
**Pull request by nikican**
I've cleaned up grammar and spelling with a bit of help from Grammarly.

### CODE REVIEW

**Review by davidnegley — APPROVED** [review]
These all look right to me, and I talk good English, so you can trust my judgment is bestest.

File: docs/fundamentals/actions.md [hunk 1]
```diff
@@ -56,49 +56,27 @@ showModule(Module.DASHBOARD);
...
 
 #### Batch actions
 
-If you find code that calls the same action multiple times right after each
-other, the payload is badly designed. This is a symptom that the callsite tries
-to use the actions as instructions for how the app should transition its state,
-rather than telling the app what the change is. In these cases, the payload 
-should be designed so it can contain, for example, changes to multiple entities
-at once.
+If you find a code that calls the same action multiple times right after each other, the payload is badly designed. This is a symptom that the call site tries to use the actions as instructions for how the app should transition its state, rather than telling the app what the change is. In these cases, the payload should be designed so it can contain, for example, changes to multiple entities at once.
```
Comment (OyvindSabo) [inline docs/fundamentals/actions.md:69]: "a code" sounds weird to me. Here I would keep code is an uncountable noun.

Comment (nikican) [reply docs/fundamentals/actions.md:69]: change to `a piece of code` | `a part of code`?

Comment (nikican) [reply docs/fundamentals/actions.md]: updated to the former

**Review by OyvindSabo — APPROVED** [review]

File: docs/fundamentals/actions.md (hunk 1 above)
Comment (davidnegley) [reply docs/fundamentals/actions.md]: I'd delete `a` as Øyvind suggested

Comment (nikican) [reply docs/fundamentals/actions.md]: Reverted to just `code`.

**Pull request merged**